sdk/keyvault/Azure.Security.KeyVault.Administration/src/RoleDefinitionType.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core;
+
+namespace Azure.Security.KeyVault.Administration.Models
+{
+    [CodeGenModel("RoleDefinitionType")]
+    public readonly partial struct KeyVaultRoleDefinitionType : IEquatable<KeyVaultRoleDefinitionType>
```

Comment:
Nit: same here and elsewhere - you don't have to "re-implement" interfaces or anything. I'd recommend not so the generator does whatever it needs to uniformly.